shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -1096,6 +1125,10 @@ export default {
           entry.config = await entry.config.save();
         }
 
+        if ( !entry.pool.hostnamePrefix ) {
```

Comment:
This is a change in behaviour for most providers (or at least for DO). 

I couldn't find anywhere it's used (set or applied) in the dashboard apart from an RKE1 use case. In `elemental-ui` there  were only references  to here.  This is definitely needed right? If so, as much as it pains me, we'll need another isElementalCluster

Reply:
@aalves08 Is this used? Created a normal RKE2 cluster and was expecting this to appear somewhere in the cluster object or machine pools, but don't see it being persisted.

Reply:
Added gate on Elemental cluster